dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/python/PythonGateway.java

```diff
@@ -232,6 +230,7 @@ public Long createOrUpdateProcessDefinition(String userName,
                                                 int releaseState,
                                                 String taskRelationJson,
                                                 String taskDefinitionJson,
+                                                String otherParamsJson,
```

Comment:
This change will fail python API submit function, maybe we should change python integrate test to avoid regression